base/atomics.jl

```diff
@@ -29,6 +29,30 @@ typealias IntTypes Union{inttypes...}
 typealias FloatTypes Union{floattypes...}
 typealias AtomicTypes Union{atomictypes...}
 
+doc"""
```

Comment:
Unless there are `$` or `\` chars that need to be escaped in the docstring this should drop the `doc` prefix since it's unnecessary. Same with the ones below.